### DISCUSSION TITLE
Resources sorting and categories

### DISCUSSION THREAD

**Discussion by mishmanners** [github/hackathons#2] [Resources]
I think the [Resources page](https://github.com/github/hackathon/blob/main/Resources.md), is okay for right now, but once if gets bigger I think we'll need a good way to sort, group, and search the page.

What do you all think?

I'm thinking categories might be a good place to start, ie:
- Virtual event platforms
- Judging tools
- Communications products
- Organiser guides

Thoughts?

**Comment by HUSK1E** [comment]
I'd be up for categories. Maybe you could group the categories alphabetically, and then have pretty headings for each one like you have in the current docs.

**Comment by arkodyuti** [comment]
@mishmanners 👋🏻 I think it'd be great to add the newly added tools like Streamyard available on [Virtual Events Kit](https://education.github.com/pack?sort=popularity&tag=Virtual+Events) collection and [Hackthon Server Template on Discord](https://discord.com/template/UqzZCTj2zfwy) 🚀

**Reply by mishmanners** [reply]
100% these will be added 😄

**Reply by mishmanners** [reply]
These ones added as they are good resources for hackathon organisers: Canva, Namecheap, Streamyard, and the Discord Hackathon Server template. Thanks for putting these on my radar.
